### DISCUSSION TITLE
Why is timezone needed for daily weather variables?

### DISCUSSION THREAD

**Discussion by drmrbrewer** [open-meteo/open-meteo#243] [Q&A]
Just curious to know why timezone needed for daily weather variables, both for [forecast](https://open-meteo.com/en/docs) and [historical](https://open-meteo.com/en/docs/historical-weather-api) APIs?  Since the basic starting point is typically just a (lat, long) coordinate, finding the timezone is an extra step.  Not a major problem as it's possible to get this by another lookup (e.g. using the [geocoding API](https://open-meteo.com/en/docs/geocoding-api)), but it adds an extra step that could be absorbed into the base API instead, for ease of use?

**Comment by patrick-zippenfenig** [comment]
`&timezone=auto` is already doing it. Technically there is no reason why the API does not per default select a timezone. From my perspective I want to ensure that developers know they have to deal with local time or UTC time.

**Reply by drmrbrewer** [reply]
Ah, thanks I hadn't spotted the possibility to use `&timezone=auto`... very good!

Though I still don't follow why setting an explicit timezone (or using `auto`) is required for daily but not also for hourly?
